### ISSUE TITLE
[Request] [X] Maps component improvements & "on touch" events

### ISSUE THREAD

**Issue by gre4twh1te** [thunkable/thunkable-issues#125]
Right now unfortunately the map component does not provide coordinates / possibility to add a marker **on touch** That really  limits the abilities for apps that ask users to provide a certain location (e.g. adding a new restaurant, reporting some local issues,...) Also there is no way to get a marker id and not even to remove a created marker.

**Describe the solution you'd like**
I want the maps component to be able to provide coordinates based on user touch and the possibility to add a marker on touch. Also I think it is essential to implement marker IDs and add possibility to remove markers once created.

**Describe alternatives you've considered**
there is no alternative solution available to this.

**Which platform are you asking for?**
Thunkable ✕

**Comment by mark-friedman** [comment]
Yeah, these are known limits of the map component and we are looking to add all the things you mentioned.

**Comment by nicbyron** [comment]
+1 for the map improvements!

**Comment by jane-d-alt** [comment]
Map now has event blocks including 'on Press' and 'on long Press', so markers can be placed where the user presses. 

Individual markers can be identified by latitude and longitude https://community.thunkable.com/t/airtable-map-show-image-when-marker-pressed/251154?u=jane

Specific marker IDs are still a good suggestion.

Will rename this issue so that the outstanding request can be better understood at a glance.
